portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,118 @@
+body {
+  background-color: aliceblue;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 700px;
+  border-left: 20px;
+}
+
+#name {
+  overflow:auto;
+  margin-bottom: 15px;
+}
+
+#name img {
+  float:left;
+  border-radius: 50%;
+  margin-right: 10px
+
+  }
+
+#name h1 {
+  margin-left: 0px;
+  margin-top:77px;
+  margin-right: 5px
+}
+
+#name p {
+  float:left;
+  margin-top:0px;
+}
+
+
+button {
+  border-radius: 4px;
+  background-image: -webkit-gradient(
+        linear,
+        right bottom,
+        left top,
+        color-stop(0, #ddd),
+        color-stop(1, #eee)
+    );
+  outline: none;
+  cursor: pointer;
+  border: none;
+  padding: 2px;
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+button:hover {
+     background-image: -webkit-gradient(
+        linear,
+        right bottom,
+        left top,
+        color-stop(0, #eee),
+        color-stop(1, #eee)
+    );
+}
+
+h1 {
+  margin-top: 30px;
+  margin-left: -20px;
 }
 
-#greeting-container {
+
+#fact {
+ margin-top: 15px;
+  background-image: linear-gradient(to left, violet, indigo, blue, green, orange, red);
+  -webkit-background-clip: text;
+  color: transparent;
+  width: 200px;
+}
+
+#personal-info img {
   margin-top: 20px;
 }
+
```

Comment:
Overall your CSS is good, but it could use some organization. I'd think through how and if you could structure this doc to make it easier to find what you're looking for.

Reply:
I tried organizing it, hope it looks better.